doc/lua_api.txt

```diff
@@ -6375,7 +6381,13 @@ an itemstring, a table or `nil`.
   or those of the hand if none are defined for this item type
 * `add_wear(amount)`
     * Increases wear by `amount` if the item is a tool, otherwise does nothing
+    * Valid `amount` range is [0,65536]
     * `amount`: number, integer
+* `add_wear_by_uses(uses)`
```

Comment:
I feel like it would make more sense if it's called `max_uses` rather than `uses`